test/views/components/InputSpec.scala

```diff
@@ -153,4 +153,95 @@ class InputSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
       }
    }
   }
+
+  "English Date Input component" must {
```

Comment:
For completeness I'd add a test for the existence of a fieldset when there are additional components on the page.